projects/plugins/jetpack/modules/search/class-jetpack-instant-search.php

```diff
@@ -161,6 +161,12 @@ protected function inject_javascript_options() {
 			$excluded_post_types = array();
 		}
 
+		$group_id = '';
```

Comment:
Perhaps `null` or `false` would be more appropriate here?

Reply:
Good point, updated.